.github/workflows/aws_tfhe_fast_tests.yml

```diff
@@ -104,11 +108,11 @@ jobs:
         uses: rtCamp/action-slack-notify@b24d75fe0e728a4bf9fc42ee217caa686d141ee8
         env:
           SLACK_COLOR: ${{ job.status }}
-          SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL }}
-          SLACK_ICON: https://pbs.twimg.com/profile_images/1274014582265298945/OjBKP9kn_400x400.png
+          SLACK_CHANNEL: ${{ env.SLACK_CHANNEL }}
+          SLACK_ICON: ${{ env.SLACK_ICON }}
```

Comment:
env is already set globally should not need to repeat

Reply:
Done.